components/organisms/Layout.js

```diff
@@ -25,6 +25,7 @@ export const Layout = ({
   locale,
   langUrl,
   breadcrumbItems,
+  active,
```

Comment:
This should be more descriptive, no one would know what "active" means just by looking at it.